docs/content/en/docs/getting-started/optional/autoscaling.md

```diff
@@ -12,8 +12,9 @@ description: >
 
 ### Cluster Autoscaler configuration in EKS Anywhere cluster spec
 
-EKS Anywhere supports autoscaling worker node groups using the [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/)'s `clusterapi` cloudProvider.
+EKS Anywhere supports autoscaling worker node groups using the [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/). The Kubernetes Cluster Autoscaler Curated Package is an image and helm chart installed via the Curated Packages Controller.
 
+The helm chart utilizes the Cluster Autoscaler binary's `clusterapi` mode to scale resources.
```

Comment:
I call out that the `clusterapi` mode is an arg we pass to the binary to scale resources. Too vague?